js/apps/filter-page/controllers/filterPageCtrl.js

```diff
@@ -38,10 +38,8 @@
 			}, 250);
 		};
 
-		const loadedCardsEvent = new $window.Event('bc-filter-cards-loaded');
-
 		const publishLoadedCardsEvent = () => {
-			document.dispatchEvent(loadedCardsEvent);
+			$rootScope.$broadcast('bc-filter-cards-loaded');
```

Comment:
`document.dispatchEvent(loadedCardsEvent);` has to be included in order to make the branch contact forms work.